scylla-gdb.py

```diff
@@ -3787,7 +3787,10 @@ def __init__(self):
         self.queues = set()
 
     def _init(self):
-        qs = std_unique_ptr(gdb.parse_and_eval('seastar::smp::_qs')).get()
+        qs = gdb.parse_and_eval('seastar::smp::_qs')
+        if qs.type.code != gdb.TYPE_CODE_PTR:
+            # older Seastar use std::unique_ptr for this variable
```

Comment:
Best mention the scylla version so we can garbage-collect these at some point, without git archeology.

Reply:
I don't think we need this old varient even now, because you get the scylla-gdb.py script together with the matching version of Scylla. But it doesn't hurt either, and we can always remove it when we want to.
I'll commit because this is (one of the things) blocking next promotion.

Reply:
Yes, but we use master's `scylla-gdb.py` with all other currently supported releases, because don't backport fixes and improvements and the `scylla-gdb.py` of those older releases is usually broken and out-of-date.

Reply:
> Yes, but we use master's `scylla-gdb.py` with all other currently supported releases, because don't backport fixes and improvements and the `scylla-gdb.py` of those older releases is usually broken and out-of-date.

Oh, I didn't know that we do that. We could have done this for a lot of other things (including tests), but we don't. Ok.